### DISCUSSION TITLE
Using custom Epic Stack templates for create-epic-app

### DISCUSSION THREAD

**Discussion by arminnaimi** [epicweb-dev/epic-stack#778] [Ideas]
Recently, I have forked the Epic Stack to create two templates to use Radix UI Themes and Mantine. 

I could not find a way to pass these templates as params to `create-epic-app`. Is there something I am overlooking? Or is this something that could be added?

**Comment by kentcdodds** [comment]
This isn't a feature (nor will it be) of create-epic-app. But you could look at what https://npm.im/create-epic-app does and make your own package to do the same thing.
